To-do list/Controllers/TaskListViewController.swift

```diff
@@ -22,110 +22,66 @@ class TaskListViewController: UIViewController, TaskTableViewCellDelegate {
         tasksTableView.delegate = self
         tasksTableView.dataSource = self
     }
-    
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
-        users = UserDefaultsManager.shared.getValueForUser() ?? [User]()
-        tasks = users[userIndex].task ?? [Task]()
-        tasksTableView.reloadData()
-    }
-    
+      
     @IBAction func addButton(_ sender: UIButton) {
-        let taskDetailsVC = self.storyboard?.instantiateViewController(
-            withIdentifier: "TaskDetailsViewController") as? TaskDetailsViewController
-        taskDetailsVC?.userIndex = userIndex
-        taskDetailsVC?.choosenTableViewCell = false
-        self.navigationController?.pushViewController(taskDetailsVC!, animated: true)
+        let taskDetailsVC = TaskDetailsViewController.loadFromStoryboard(
+            type: TaskDetailsViewController.self)
+        taskDetailsVC.id = tasks.count+1
```

Comment:
create empty Task with new id instead of passing just `id`

Reply:
done